server/src/main/java/com/genymobile/scrcpy/Server.java

```diff
@@ -188,6 +219,9 @@ private static Options createOptions(String... args) {
         boolean powerOffScreenOnClose = Boolean.parseBoolean(args[15]);
         options.setPowerOffScreenOnClose(powerOffScreenOnClose);
 
+        EnumSet<Intents> broadcastIntents = Intents.fromBitSet(BitSet.valueOf(new long[]{Long.parseLong(args[16])}));
```

Comment:
This is a roundabout way of translating but it is the way that seemed to be most direct without implementing manually